composer/spec/dependabot/composer/file_updater_spec.rb

```diff
@@ -113,22 +102,38 @@
     end
 
     describe "the updated lockfile" do
-      subject(:updated_lockfile_content) do
+      let(:updated_lockfile_content) do
         updated_files.find { |f| f.name == "composer.lock" }.content
       end
+      let(:parsed_updated_lockfile_content) { JSON.parse(updated_lockfile_content) }
+      let(:updated_lockfile_entry) do
+        parsed_updated_lockfile_content["packages"].find do |package|
+          package["name"] == dependency.name
+        end
+      end
+
+      it "updates the dependency version and plugin-api-version (to match instaled composer) in the lockfile" do
+        expect(updated_lockfile_entry["version"]).to eq("1.22.1")
+        expect(parsed_updated_lockfile_content["prefer-stable"]).to be(false)
```

Comment:
❓ Is this just a feature of v2 files or are we explicitly testing we are going from 'version unspecified to v2'?

Reply:
Don't actually know what `prefer-stable` is 🤔 copy pasted from the previous spec but added the spec to check the `plugin-api-version` matches the installed composer version (updates from 2.1.0 back to 2.0.0)